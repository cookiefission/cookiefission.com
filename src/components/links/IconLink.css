.IconLink {
  color: black;
}

.IconLink:hover {
  color: green;
}
