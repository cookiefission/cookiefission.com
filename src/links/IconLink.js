import React from 'react';
import Icon from '../Icon';

const IconLink = (props) => {
  const href = props.href;
  const iconName = props.iconName;

  return(
    <a href={href} target='blank'>
      <Icon name={iconName} />
    </a>
  );
};

export default IconLink;
