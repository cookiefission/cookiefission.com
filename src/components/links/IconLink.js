import React from 'react';
import Icon from '../Icon';

import './IconLink.css';

const IconLink = (props) => {
  return(
    <a href={props.href} target='blank' className='IconLink' >
      <Icon name={props.iconName} />
    </a>
  );
};

export default IconLink;
