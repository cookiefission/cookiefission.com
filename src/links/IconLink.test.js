import React from 'react';
import IconLink from './IconLink';
import renderer from 'react-test-renderer';

describe('<IconLink />', () => {
  it('matches the snapshot', () => {
    const component = renderer.create(
      <IconLink href='https://example.com' />
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
