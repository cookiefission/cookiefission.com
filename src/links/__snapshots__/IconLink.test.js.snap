exports[`<IconLink /> matches the snapshot 1`] = `
<a
  href="https://example.com"
  target="blank">
  <div
    className="Icon-wrapper">
    <i
      className="fa fa-warning fa-lg" />
  </div>
</a>
`;
